<ui version="4.0" >
 <class>paisbase</class>
 <widget class="QWidget" name="paisbase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Paises</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_centroCoste.xpm</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_gestortareas.48x48.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>11</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QToolButton" name="mui_crear" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Nuevo pais</string>
          </property>
          <property name="whatsThis" >
           <string>Nuevo pais</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+N</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar pais</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar pais</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_editDelete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="mui_guardar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar pais</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar pais</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_save.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4" >
         <widget class="QToolButton" name="mui_detalles" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Ver/ocultar datos pais</string>
          </property>
          <property name="whatsThis" >
           <string>Ver/ocultar datos pais</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_tuxPeq.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+B</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>171</width>
            <height>34</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTableWidget" name="mui_list" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Datos del pais:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Nombre del pais:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="mui_descpais" >
        <property name="maxLength" >
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Codigo 2 letras:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mui_cod2pais" >
        <property name="maxLength" >
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel1_3" >
        <property name="text" >
         <string>Codigo 3 letras:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mui_cod3pais" >
        <property name="maxLength" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>191</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2" >
       <property name="text" >
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>paisbase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>548</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel" >
     <x>549</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mui_detalles</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel" >
     <x>114</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
